<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FEM Draw</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/icon.qrc">
    <normaloff>:/resources/icon.ico</normaloff>:/resources/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>150</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QCheckBox" name="checkBox_Color">
           <property name="toolTip">
            <string>Draw color image</string>
           </property>
           <property name="text">
            <string>Color</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_Color">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Data component for colors and isolines</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QCheckBox" name="checkBox_Isolines">
           <property name="toolTip">
            <string>Draw isolines</string>
           </property>
           <property name="text">
            <string>Isolines</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_Isolines">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Number of isolines</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCheckBox" name="checkBox_Vectors">
           <property name="toolTip">
            <string>Draw vectors</string>
           </property>
           <property name="text">
            <string>Vectors</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_Vectors">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Draw vectors for every i value</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>U:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_Vectors_U">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Data component for first vector axis</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>V:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_Vectors_V">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Data component for second vector axis</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="paintwidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>640</width>
      <height>250</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Tecplot_File"/>
    <addaction name="separator"/>
    <addaction name="actionTransparent_Image"/>
    <addaction name="actionCopy_Image_to_Clipboard"/>
    <addaction name="actionSave_Image_File"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Window"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuInterpolation">
    <property name="title">
     <string>Interpolation</string>
    </property>
    <addaction name="actionIncrease_Interpolation"/>
    <addaction name="actionDecrease_Interpolation"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_FEM_Draw"/>
    <addaction name="actionAbout_Third_Party_Libraries"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Configuration</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionEnglish"/>
     <addaction name="actionRussian"/>
    </widget>
    <addaction name="menuLanguage"/>
    <addaction name="actionShow_Legend"/>
    <addaction name="actionUse_Light_Colors"/>
    <addaction name="actionUse_Purple_Colors"/>
    <addaction name="actionMemory_Limit"/>
   </widget>
   <widget class="QMenu" name="menuIsolines">
    <property name="title">
     <string>Isolines</string>
    </property>
    <addaction name="actionShow_Isolines"/>
    <addaction name="actionIsolines_Color"/>
    <addaction name="actionIsolines_Width"/>
   </widget>
   <widget class="QMenu" name="menuVectors">
    <property name="title">
     <string>Vectors</string>
    </property>
    <addaction name="actionShow_Vectors"/>
    <addaction name="actionVectors_Color"/>
    <addaction name="actionVectors_Width"/>
    <addaction name="actionVectors_Length"/>
    <addaction name="actionArrowSize"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuInterpolation"/>
   <addaction name="menuIsolines"/>
   <addaction name="menuVectors"/>
   <addaction name="menuConfiguration"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionOpen_Tecplot_File">
   <property name="text">
    <string>Open Tecplot File...</string>
   </property>
   <property name="toolTip">
    <string>Open Tecplot File...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCopy_Image_to_Clipboard">
   <property name="text">
    <string>Copy Image to Clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionSave_Image_File">
   <property name="text">
    <string>Save Image File...</string>
   </property>
   <property name="toolTip">
    <string>Save Image File...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNew_Window">
   <property name="text">
    <string>New Window</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionIncrease_Interpolation">
   <property name="text">
    <string>Increase Interpolation</string>
   </property>
   <property name="toolTip">
    <string>Increase number of interpolation (bilinear)</string>
   </property>
   <property name="shortcut">
    <string>=</string>
   </property>
  </action>
  <action name="actionDecrease_Interpolation">
   <property name="text">
    <string>Decrease Interpolation</string>
   </property>
   <property name="toolTip">
    <string>Decrease number of interpolation (bilinear)</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionAbout_FEM_Draw">
   <property name="text">
    <string>About FEM Draw...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionAbout_Third_Party_Libraries">
   <property name="text">
    <string>About Third Party Libraries...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionTransparent_Image">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Transparent Image</string>
   </property>
  </action>
  <action name="actionShow_Isolines">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Isolines</string>
   </property>
  </action>
  <action name="actionIsolines_Color">
   <property name="text">
    <string>Isolines Color...</string>
   </property>
  </action>
  <action name="actionIsolines_Width">
   <property name="text">
    <string>Isolines Width...</string>
   </property>
  </action>
  <action name="actionShow_Vectors">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Vectors</string>
   </property>
  </action>
  <action name="actionVectors_Color">
   <property name="text">
    <string>Vectors Color...</string>
   </property>
  </action>
  <action name="actionVectors_Width">
   <property name="text">
    <string>Vectors Width...</string>
   </property>
  </action>
  <action name="actionVectors_Length">
   <property name="text">
    <string>Vectors Length...</string>
   </property>
  </action>
  <action name="actionArrowSize">
   <property name="text">
    <string>Vectors Arrow Size...</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="actionShow_Legend">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Legend</string>
   </property>
  </action>
  <action name="actionUse_Light_Colors">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use Light Colors</string>
   </property>
  </action>
  <action name="actionUse_Purple_Colors">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use Purple Colors</string>
   </property>
  </action>
  <action name="actionMemory_Limit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1 GiB Memory Limit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>paintwidget</class>
   <extends>QWidget</extends>
   <header>paintwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/icon.qrc"/>
 </resources>
 <connections/>
</ui>
